tests/test_tutorial/test_fastapi/test_delete/test_tutorial001.py

```diff
@@ -210,7 +210,10 @@
                 "properties": {
                     "name": {"title": "Name", "type": "string"},
                     "secret_name": {"title": "Secret Name", "type": "string"},
-                    "age": {"title": "Age", "type": "integer"},
+                    "age": {
+                        "title": "Age",
+                        "anyOf": [{"type": "integer"}, {"type": "null"}],
```

Comment:
Not sure if this is intended, as the real DB model is not nullable often. TBD

Reply:
I see. I believe this is caused by Pydantic v1 confusion between nullable and optional fields (see [changes-to-json-schema-generation](https://docs.pydantic.dev/2.0/migration/#changes-to-json-schema-generation) and [required-optional-and-nullable-fields](https://docs.pydantic.dev/2.0/migration/#required-optional-and-nullable-fields)). I think it makes sense to align to Pydantic v2 way of specifying nullable and optional fields.